Doc/tools/extensions/pyspecific.py

```diff
@@ -34,7 +34,7 @@
 
 
 ISSUE_URI = 'https://bugs.python.org/issue%s'
-SOURCE_URI = 'https://hg.python.org/cpython/file/3.6/%s'
+SOURCE_URI = 'https://github.com/python/cpython/tree/3.6/%s'
```

Comment:
My patch for another issue will address this, I think 🤔 

http://bugs.python.org/file45904/issue28941.patch